bootloader-formula/bootloader/macros.jinja

```diff
@@ -0,0 +1,42 @@
+{#-
+Jinja macros file for the Bootloader Salt states
+Copyright (C) 2023 Georg Pfuetzenreuter <mail+opensuse@georg-pfuetzenreuter.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- set boolmap = {'default': {true: 'true', false: 'false'}, 'sysconfig': {true: 'yes', false: 'no'}} %}
+
+{%- macro kvconfig(name, type, config) -%}
+{%- if type == 'sysconfig' %}
+{%- set file = '/etc/sysconfig/' ~ name %}
+{{ name }}_sysconfig:
+{%- elif type == 'default' %}
+{%- set file = '/etc/default/' ~ name %}
+{{ name }}_default:
+{%- endif %}
```

Comment:
I'd recommend to add a `{% else %}` that warns about unknown/unhandled `type` values.

Reply:
I thought about it as well but figured it to be superfluous since it's only used here and will fail shortly after when `type` is referenced. Added an error message now.